lib/open_project/custom_styles/color_themes.rb

```diff
@@ -33,16 +33,18 @@ module ColorThemes
     OpenProject::CustomStyles::ColorThemes::DEFAULT_THEME_NAME = "OpenProject".freeze
 
     DEPRECATED_ALTERNATIVE_COLOR = "#35C53F".freeze
+    DEPRECATED_BIM_ALTERNATIVE_COLOR = "#349939".freeze
```

Comment:
It took me a while to find where this got introduced. It might make sense to reference this constant also in the [migration that introduces the value](https://github.com/opf/openproject/blob/dev/modules/bim/db/migrate/20201105154216_seed_custom_style_with_bim_theme.rb#L87)